### DISCUSSION TITLE
flask init db could not find locally referenced modules

### DISCUSSION THREAD

**Discussion by eavf** [miguelgrinberg/Flask-Migrate#503] [Q&A]
I am experiencing repeatedly, I suppose, a stupid problem. When I want to create initial migration of my models to the mysql database hosted on a local network server, I got the following error:

`flask db init
Error: While importing 'flaskGPT.app', an ImportError was raised:

Traceback (most recent call last):
  File "C:\Users\eavf\.virtualenvs\flaskGPT-yypoX3qF\lib\site-packages\flask\cli.py", line 218, in locate_app
    __import__(module_name)
  File "C:\Users\eavf\PycharmProjects\flaskGPT\app.py", line 2, in <module>
    from formulare import ForgotPasswordForm, LoginForm, RegisterForm
ModuleNotFoundError: No module named 'formulare'

Usage: flask [OPTIONS] COMMAND [ARGS]...
Try 'flask --help' for help.

Error: No such command 'db'.
`

I have to say, that : 
1. file formulare.py is in the same directory as app.py and where flask init db commands were run.
2. pycharm does not see an error, missing reference on this file. 
3. the problem is with all locally referred modules. 
4. I am no windows

**Comment by miguelgrinberg** [comment]
Does `python -m flask db init` work? This looks like an import path configuration issue to me, not related at all to Flask-Migrate, but to the `flask` command in general.
